examples/mongodbatlas_advanced_cluster/effective_fields/existing_module/module_v1/outputs.tf

```diff
@@ -0,0 +1,57 @@
+output "project_id" {
+  description = "Atlas project id"
+  value       = mongodbatlas_project.this.id
+}
+
+output "project_name" {
+  description = "Atlas project name"
+  value       = mongodbatlas_project.this.name
+}
+
+output "cluster_id" {
+  description = "Atlas cluster id"
+  value       = mongodbatlas_advanced_cluster.this.cluster_id
+}
+
+output "cluster_name" {
+  description = "Atlas cluster name"
+  value       = mongodbatlas_advanced_cluster.this.name
+}
+
+output "cluster_state" {
+  description = "Atlas cluster state"
+  value       = mongodbatlas_advanced_cluster.this.state_name
+}
+
+output "connection_strings" {
+  description = "Atlas cluster connection strings"
+  value       = mongodbatlas_advanced_cluster.this.connection_strings
+  sensitive   = true
+}
+
+# In v1, we only expose configured specifications
+# We cannot see the actual (effective) values that Atlas has provisioned after auto-scaling
+# This is a limitation of the ignore_changes approach
+output "configured_specs" {
+  description = "Configured hardware specifications as defined in Terraform configuration"
```

Comment:
I see this related to my previous comment, so when using ignore_changes (which allows difference between config and state) when you reference a value from the resource it returns the value of the config?